src/tests/issues.targets

```diff
@@ -3648,6 +3648,13 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Known failures for mono runtime on android-x64 -->
+    <ItemGroup Condition="('$(RuntimeFlavor)' == 'mono' and $(TargetOS)' == 'android' and '$(TargetArchitecture)' == 'x64'">
```

Comment:
I was thinking of disabling this for all of android instead of just x64. Are we sure this is limited to x64? That would seem odd.

Reply:
The android-arm64 tests are disabled https://github.com/dotnet/runtime/issues/60128. It might be safer to disable for all android.